vignettes/web_only/BCRpipeline.Rmd

```diff
@@ -103,7 +103,7 @@ Each clonal lineage has its own **germline sequence** that represents the ancest
 
 In Immunarch, `repGermline()` function generates germline for each sequence:
 
-```{r example 3}
+```{r example 3, results = 'hide'}
 #generate germline
 bcrdata$data %>%
     top(5) %>%
```

Comment:
You can remove top(5) if you add results = 'hide', because top() was added to avoid displaying too large result

Reply:
So, remove `top(5) %>%` line, because it's not needed with hide